### DISCUSSION TITLE
help me pls

### DISCUSSION THREAD

**Discussion by RodjAI** [termux/termux-packages#11299] [Q&A] [answered]
$ pkg update && pkg upgrade -y
Testing the available mirrors:
[*] https://dl.bintray.com/termux/termux-packages-24: bad
[*] https://grimler.se/termux-packages-24: ok
[*] https://main.termux-mirror.ml: ok
[*] https://termux.mentality.rip/termux-packages-24: ok
Picking mirror: https://grimler.se/termux-packages-24
Ign:1 https://dl.bintray.com/grimler/game-packages-24 games InRelease
Get:2 https://grimler.se/termux-packages-24 stable InRelease [16.8 kB]
Ign:3 https://dl.bintray.com/grimler/science-packages-24 science InRelease
Err:4 https://dl.bintray.com/grimler/game-packages-24 games Release
  OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to dl.bintray.com:443
Err:5 https://dl.bintray.com/grimler/science-packages-24 science Release
  OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to dl.bintray.com:443
Err:2 https://grimler.se/termux-packages-24 stable InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B0076E490B71616B
Reading package lists... Done
E: The repository 'https://dl.bintray.com/grimler/game-packages-24 games Release' does not have a Release file.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).
E: The repository 'https://dl.bintray.com/grimler/science-packages-24 science Release' does not have a Release file.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).
W: GPG error: https://grimler.se/termux-packages-24 stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B0076E490B71616B
E: The repository 'https://grimler.se/termux-packages-24 stable InRelease' is not signed.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list

**Comment by sylirre** [comment, marked as answer]
Google Play Store distribution source is deprecated and shouldn't be used anymore.

You'll need to uninstall it and proceed with installation instructions shown there: https://github.com/termux/termux-app#installation
